viewto-frontend/src/models/ReferenceObject.ts

```diff
@@ -33,17 +31,26 @@ export default class ReferenceObject {
     data: any = undefined;
 
     @action
-    setData(data: any)  {
+    setData(data: any) {
         this.data = data;
     }
 
+    @observable
+    referenceObject: string = "";
+
+    @action
+    setReference(data: string) {
+        this.referenceObject = data;
+    }
+
     @action
     async load() {
         if (!this.isLoading) {
             const { speckle, project } = (stores as Stores).scenario;
             this.setIsLoading();
             const obj = await speckle.Project(project.id).Version(this.id).get;
             this.setData(obj);
+            this.setReference(obj.referencedObject);
```

Comment:
This is duplicate state. We should either get this value from `data` or create a computed property for this value that points to `data.referencedObject`.

Reply:
made these changes to computed